CDP4ObjectBrowser/AutoGen/ReferenceDataLibraryRowViewModel.cs

```diff
@@ -106,7 +106,7 @@ protected ReferenceDataLibraryRowViewModel(T referenceDataLibrary, ISession sess
             this.constantFolder = new CDP4Composition.FolderRowViewModel("Constant", "Constant", this.Session, this);
             this.ContainedRows.Add(this.constantFolder);
 
-            var rdlListener = this.messageBus.Listen<SessionEvent>()
+            var rdlListener = this.CDPMessageBus.Listen<SessionEvent>()
```

Comment:
this is generated, did you update the generator?

Reply:
Issue created to refactor T4 scripts: #1309